src/editorManager.js

```diff
@@ -5,441 +5,519 @@ import { getEditorInstance } from './editors';
 import EventManager from './eventManager';
 import { EditorState } from './editors/_baseEditor';
 
-function EditorManager(instance, priv, selection) {
-  const _this = this;
-  const eventManager = new EventManager(instance);
-  let destroyed = false;
-  let lock = false;
-  let activeEditor;
+class EditorManager {
+  /**
+   * @param {Handsontable} instance
+   * @param {GridSettings} priv
+   * @param {Selection} selection
+   */
+  constructor(instance, priv, selection) {
+    /**
+     * Instance of {@link Handsontable}
+     *
+     * @private
+     * @type {Handsontable}
+     */
+    this.instance = instance;
+    /**
+     * Reference to an instance's private GridSettings object.
+     *
+     * @private
+     * @type {GridSettings}
+     */
+    this.priv = priv;
+    /**
+     * Instance of {@link Selection}
+     *
+     * @private
+     * @type {Selection}
+     */
+    this.selection = selection;
+    /**
+     * Instance of {@link EventManager}.
+     *
+     * @private
+     * @type {EventManager}
+     */
+    this.eventManager = new EventManager(instance);
+    /**
+     * Determines if EditorManager have destroyed.
+     *
+     * @private
+     * @type {Boolean}
+     */
+    this.destroyed = false;
+    /**
+     * Determines if EditorManager have locked.
```

Comment:
Please change from "Determines if EditorManager have locked" to "Determines if EditorManager has been locked"